dom/events/enter/enter.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+var domEvents = require("../events");
+var singleReference = require("../../../js/single-reference/single-reference");
+var CID = require("../../../js/cid/get-cid");
+
+/**
+ * @module {events} can-util/dom/events/enter/enter enter
+ * @parent can-util/dom/events/events
+ *
+ * Watches for when enter keys are pressed on a given element
+ *
+ * ```js
+ * var events = require("can-util/dom/events/events");
+ * var input = document.createElement("input");
+ *
+ * function enterEventHandler() {
+ * 	console.log("enter key pressed");
+ * }
+ *
+ * events.addEventHandler.call(input, "enter", enterEventHandler);
+ * events.dispatch.call(input, {
+ *   type: 'keyup',
+ *   keyCode: keyCode
+ * });
+ *
+ *
+ */
+
+var origAddEvt = domEvents.addEventListener;
+domEvents.addEventListener = function(eventName, handler){
+   if(eventName === "enter") {
+       var boundHandler = function(ev){
+           if (ev.keyCode === 13 || ev.key === "Enter") {
```

Comment:
can-stache-bindings uses multiple ways of simulating an enter event here: https://github.com/canjs/can-stache-bindings/blob/master/test/bindings-define-test.js#L239

The tests should probably use this method so both ways work.  This is why the code is checking `ev.keyCode` and `ev.key`.  The tests should hit those cases.